src/components/message/message.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
+ *
+ * @TODO
+ * 1. Style `:focus` state
+ */
+
+.c-message {
+  align-items: center; /* 1 */
+  background: colors.$gray-light;
+  border-top: 4px solid colors.$gray-dark;
+  color: colors.$text-dark;
+  display: flex; /* 1, 2 */
+  justify-content: space-between; /* 2 */
+  line-height: ms.step(1, 1); /* 3 */
+  padding: ms.step(-8) ms.step(0);
+}
+
+.c-message--warning {
+  background: colors.$fuchsia;
+  border-top: 4px solid colors.$fuchsia-darker;
+  color: colors.$text-light-emphasis;
+}
+
+/**
+ * Inherit parent text color for links when blue is hideous against background.
+ */
+
+.c-message--warning a {
+  color: inherit;
+}
+
+/**
+ * 1. Reset default button styles (remove once reset/button CSS is inplace)
+ * 2. Close icon should be the same color as the parent text color.
```

Comment:
Did you already try using `c-button c-button--link`?

Reply:
🤔 I did not see a `c-button--link` or related selector in `button.scss`.

Reply:
Sorry, I meant `c-button--tertiary`.

Reply:
Example of (close) button with `c-button--tertiary`

![Screen Shot 2020-04-13 at 3 24 18 PM](https://user-images.githubusercontent.com/1427548/79167808-30c86b80-7d9d-11ea-9ff3-c243fe31c89d.png)

Example of (close) button with `c-message__close`

![Screen Shot 2020-04-13 at 3 24 38 PM](https://user-images.githubusercontent.com/1427548/79167979-9fa5c480-7d9d-11ea-8b50-be0d4ff0761d.png)